### ISSUE TITLE
Pre install registry rake task? 

### ISSUE THREAD

**Issue by zaggash** [EugenMayer/docker-image-portus#13]
In the readme, you  comment this : 

adding a new rake task to pre-install a registry, should be removed when https://github.com/SUSE/Portus/issues/1036 is finished

The task is closed, isn't it?

**Comment by EugenMayer** [comment]
Yes, we now need to migrate the bootstrap script to use that core task, i remember there was an issue with missing arguments or similar, but i cant quiet make it up

**Comment by zaggash** [comment]
Ok, I just wanted to rise the question. 
Nice work on the setup tho. 
I'm using it in production for the record :)

**Comment by EugenMayer** [comment]
Thanks, actually this all is used with rancher in https://github.com/EugenMayer/docker-rancher-extra-catalogs/tree/master/templates/registry-slim/19 and we run int there as production
]
I will need to apply your fixes to https://github.com/EugenMayer/docker-rancher-extra-catalogs/blob/master/templates/registry-slim/19/docker-compose.yml#L24 and https://github.com/EugenMayer/docker-rancher-extra-catalogs/blob/master/templates/registry-slim/19/docker-compose.yml#L43
